### ISSUE TITLE
update readme about evnets

### ISSUE THREAD

**Issue by xuanjiazhen** [gizak/termui#191]
![image](https://user-images.githubusercontent.com/41418088/46779514-c28b8e80-cd4a-11e8-9f8a-faae7c26cd67.png)

the description  for events in readme is  outdated after the rework .
`Event.Data`  no longer exists in the current version

**Comment by cjbassi** [comment]
You're right and that's something I'm aware of. My thinking was that the README should reflect the latest release of termui where events have not been reworked. But if someone wanted to use the latest commits to master, they would be free to do so but at their own risk.

I'm open to feedback on this though, I wasn't sure what the best strategy of making breaking changes was but this seemed fine to me.

**Comment by mou** [comment]
Best strategy is to make Readme content relevant to revision of code associated with it. Eg. when you switch branches content of Readme updates according to source code in this revision

**Comment by cjbassi** [comment]
Hi, I figured updating the readme to the latest commit was the best decision so I went ahead and did that and I also added a wiki with some general info. Let me know if anyone has any more suggestions.
